packages/notification-config-process/src/routers/NotificationConfigRouter.ts

```diff
@@ -23,16 +31,75 @@ const notificationConfigRouter = (
     .router(notificationConfigApi.processApi.api, {
       validationErrorHandler: zodiosValidationErrorToApiProblem,
     })
-    .post("/tenantNotificationConfigs", async (req, res) => {
+    .get("/tenantNotificationConfigs", async (req, res) => {
       const ctx = fromAppContext(req.ctx);
 
       try {
         validateAuthorization(ctx, [ADMIN_ROLE]);
-        await notificationConfigService.updateTenantNotificationConfig(
-          req.body,
+        const tenantNotificationConfig =
+          await notificationConfigService.getTenantNotificationConfig(ctx);
+        return res
+          .status(200)
+          .send(
+            notificationConfigApi.TenantNotificationConfig.parse(
+              tenantNotificationConfigToApiTenantNotificationConfig(
+                tenantNotificationConfig
+              )
+            )
+          );
+      } catch (error) {
+        const errorRes = makeApiProblem(
+          error,
+          getTenantNotificationConfigErrorMapper,
+          ctx
+        );
+        return res.status(errorRes.status).send(errorRes);
+      }
+    })
+    .get("/userNotificationConfigs", async (req, res) => {
+      const ctx = fromAppContext(req.ctx);
+
+      try {
+        validateAuthorization(ctx, [ADMIN_ROLE, API_ROLE]);
+        const userNotificationConfig =
+          await notificationConfigService.getUserNotificationConfig(ctx);
+        return res
+          .status(200)
+          .send(
+            notificationConfigApi.UserNotificationConfig.parse(
+              userNotificationConfigToApiUserNotificationConfig(
+                userNotificationConfig
+              )
+            )
+          );
+      } catch (error) {
+        const errorRes = makeApiProblem(
+          error,
+          getUserNotificationConfigErrorMapper,
           ctx
         );
-        return res.status(204).send();
+        return res.status(errorRes.status).send(errorRes);
+      }
+    })
+    .post("/tenantNotificationConfigs", async (req, res) => {
+      const ctx = fromAppContext(req.ctx);
+
+      try {
+        validateAuthorization(ctx, [ADMIN_ROLE]);
+        const tenantNotificationConfig =
+          await notificationConfigService.updateTenantNotificationConfig(
```

Comment:
queste al momento sono delle upsert, ma sarà da capire se separare creazione e update in due operazioni diverse